api-controllers/ServiceFabrikAdminController.js

```diff
@@ -403,11 +398,12 @@ class ServiceFabrikAdminController extends FabrikBaseController {
           .getServiceInstances(`service_plan_guid IN ${guids}`)
           .map(instance => {
             const plan = getPlanByGuid(plans, instance.entity.service_plan_guid);
-            return this
-              .createManager(plan.id)
-              .then(manager => _
+            return Promise.try(() => DirectorService.createInstance(_.get(instance, 'metadata.guid'), {
+                plan_id: plan.id
+              }))
+              .then(service => _
                 .chain(instance)
-                .set('manager', manager)
+                .set('serviceInstance', service)
```

Comment:
can we not name it `serviceInstance` ? This naming somewhat conflicts with the generally used `serviceInstance` and even though i understand you intend to mean instance of Directorservice, just while reading through code at first its not easy to understand it. Especially when i read it as `depoyment.serviceInstance.getNetworkSegmentIndex(deployment.name)` in the above. Can it rather be renamed just as `service` or `directorService` so that its more explicit and we know which impl it refers to?